### ISSUE TITLE
Use libnotify for challenge codes

### ISSUE THREAD

**Issue by RainbowAsteroids** [trustcrypto/onlykey-agent#31]
When you use `onlykey-gpg-agent`, the challenge codes are outputted to the terminal, which makes it inconvientent to alt-tab to another terminal to use GPG. It would help the UX if these codes were provided through a notifcation.

**Comment by onlykey** [comment]
Yes, this is one of the things we still want to implement. We are looking for a way to support this on both Linux and Mac though, eventually Windows too.
